src/main/java/com/epam/university/java/core/task003/FlatMappingOperationImpl.java

```diff
@@ -0,0 +1,38 @@
+package com.epam.university.java.core.task003;
+
+public class FlatMappingOperationImpl implements FlatMappingOperation {
+    @Override
+    public String[] flatMap(String source) {
+        int sizeOfOutput = 0;
+        String outputStr = "";
+        for (int i = 0; i < source.length(); i++) {
+            if (source.charAt(i) >= '0' && source.charAt(i) <= '9') {
```

Comment:
It's not a good idea to rely on characters here. It is slightly better to split source string with delimiter and then trim parts.